### PR TITLE
fix: Fixed typos found in documentations

### PR DESCRIPTION
**Pull request by covalentbond**
Corrected some typos & corrected some spellings

### CODE REVIEW

**Comment by covalentbond** [comment]
@googlebot I signed it!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ just a few small guidelines you need to follow.
 
 1. File a bug at https://github.com/google/shaka-player/issues (if there isn't
    one already).  If your patch is going to be large, you should start a
-   discussion on the mailing list first to make sure it is in line with our
+   discussion on the [mailing list][] first to make sure it is in line with our
```
Comment (joeyparrish) [inline CONTRIBUTING.md:8]: I appreciate you trying to add a useful link, but this information is actually out of date.  We no longer have a discussion-type list, but instead ask for questions to be filed on GitHub issues.  The shaka-player-users list was changed a couple years ago into a list where only the team can post, for us to send out release announcements and developer surveys.

Comment (covalentbond) [reply CONTRIBUTING.md:8]: Shall we update the docs by removing the statement instead. I think, it will bring more clarity.

Comment (joeyparrish) [reply CONTRIBUTING.md:8]: Yes, please.  You can say "start a discussion on GitHub first" instead.  Thank you!

**Review by joeyparrish — APPROVED** [review]

**Comment by joeyparrish** [comment]
I'm very sorry.  I made a mistake when I merged your pull request.  (I'm very tired today.)

When I read your pull request, I somehow only read the first file.  So when I merged it, I write a commit message that only reflected the changes in that one file.

Your other fixes are very welcome, too, and I'm sorry I wasn't more attentive when I reviewed and merged this.

Thanks for contributing!

**Pull request merged**